src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -627,7 +627,7 @@ trait Implicits {
             loop(restpe, pt)
           else pt match {
             case tr @ TypeRef(pre, sym, args) =>
-              if (sym.isAliasType) loop(tp, pt.dealias)
+              if (sym.isAliasType) loop(tp, pt.normalize)
```

Comment:
Maybe add a comment... `normalize` is not exactly everbody's darling :-) but since we know here that it's a `TypeRef` and an `AliasType`, LGTM.